src/utils/components/GuidedTour/utils/constants.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import { Step } from 'react-joyride';
+
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { t } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+
+import AddVolumeContent from '../components/AddVolumeContent/AddVolumeContent';
+import EndTourContent from '../components/EndTourContent/EndTourContent';
+
+export const tourSteps: Step[] = [
+  {
+    content: t(
+      'Learn more about key areas to complete workflows, increase productivity, and familiarize yourself with Virtualization using our resources.',
+    ),
+    data: {
+      route: '/k8s/all-namespaces/virtualization-overview',
+    },
+    disableBeacon: true,
+    placement: 'bottom',
+    target: '#tour-step-resources',
+    title: t('Getting started resources'),
+  },
+  {
+    content: t(
+      'To create virtual machines in a project, you must first create a new project and become the administrator.',
+    ),
+    data: { route: '/k8s/all-namespaces/virtualization-overview' },
+    disableBeacon: true,
+    placement: 'right',
+    target: '.co-namespace-dropdown',
```

Comment:
better to aim to an ID , classname can be changed any time

Reply:
This is console namespace bar, it doesn't have ID, nor can I assign one